docs/workflows.rst

```diff
@@ -47,6 +49,14 @@ warp to the MNI space.
 
     Animation showing T1 to MNI normalization (ANTs)
 
+Additionally, FreeSurfer surfaces are reconstructed from T1-weighted structural
+image(s), using the ANTs-extracted brain mask.
+This feature may be disabled with the ``--no-freesurfer`` flag.
+
+.. figure:: _static/reconall.svg
+
+    Surface reconstruction (FreeSurfer)
```

Comment:
Can someone try building the docs on their machine? This one is cropped, unlike the others. It may be a consequence of the new figure generation. (The missing `:scale:` line doesn't make any difference for me.)

On that topic, we should probably update the sample report. What subject were you running it on? [`ds000005_R2.0.1`](https://openfmri.org/dataset/ds000005/)`/sub-01`? I'll run it with ANTS + FreeSurfer today and copy the new report.